File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.js
function solution(sizes) {
    var fir = [];
    var sec = [];
    for(let i=0; i<sizes.length;i++){
        if(sizes[i][0]<sizes[i][1]){
            fir.push(sizes[i][1]);
            sec.push(sizes[i][0]);
        }
        else{
            fir.push(sizes[i][0]);
            sec.push(sizes[i][1]);
        }
    }
    let answer = Math.max(...fir)*Math.max(...sec);
    return answer;
}

File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.java
class Solution {
    public int solution(int[][] sizes) {
        int answer = 0;
        int r = 0;
        int c = 0;
        
        for(int i=0; i<sizes.length; i++){
            int f = sizes[i][0];
            int s = sizes[i][1];
            
            if(s>f){
                int temp = f;
                f = s;
                s = temp;
            }
            
            if(r<f){
                r = f;
            }
            if(c<s){
                c = s;
            }
        }
        answer = r*c;
        
        return answer;
    }
}